### PR TITLE
Add support for SOFAR 2200TL

### PR DESCRIPTION
**Pull request by darshkpatel**
I tried using this library for my SOFAR inverter and noticed it was failing due to a space while parsing integer for the rated power substring it tries to read from the `myDeviceArray` string.

I have fixed that issue, updated the test to test this edge case too, and also updated the documentation adding support for SOFAR 2200TL🥂 

**Error Log for reference:**
``` py
  File "/usr/local/lib/python3.10/site-packages/omnikinverter/omnikinverter.py", line 184, in inverter
    return Inverter.from_js(data)
  File "/usr/local/lib/python3.10/site-packages/omnikinverter/models.py", line 177, in from_js
    solar_rated_power=get_value(4),
  File "/usr/local/lib/python3.10/site-packages/omnikinverter/models.py", line 158, in get_value
    return int(matches[position])
ValueError: invalid literal for int() with base 10: '2 00'
```

### CODE REVIEW

File: omnikinverter/models.py [hunk 1]
```diff
@@ -155,7 +155,8 @@ def get_value(position: int) -> Any:
                 if matches[position] != "":
                     if position in [4, 5, 6, 7]:
                         if position in [4, 5]:
-                            return int(matches[position])
+                            # Need to remove spaces before parsing int for SOFAR 2200TL
```
Comment (klaasnicolaas) [inline omnikinverter/models.py:158]: You can leave out the comment here

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Could you make a separate test with its own data fixture?

File: omnikinverter/models.py (hunk 1 above)
Comment (darshkpatel) [reply omnikinverter/models.py]: Removed

**Comment by darshkpatel** [comment]
> Could you make a separate test with its own data fixture?

Done in [b184a28](https://github.com/klaasnicolaas/python-omnikinverter/pull/311/commits/b184a28585b42149c9c0cd9040e891e9a154e9e7)

**Review by klaasnicolaas — APPROVED** [review]
